chamber/const.py

```diff
@@ -72,6 +72,8 @@
 A_COEF = (-1.6302041e-1, 1.8071570e-3, -6.7703064e-6, 8.5813609e-9)
 B_COEF = (-5.9890467e1, 3.4378043e-1, -7.7326396e-4, 6.3405286e-7)
 
+#Gravitational acceleration 
+GRAVIT_ACCLR = 9.80665 #m/s^2 
```

Comment:
I am not sure if you have a linter installed. It would point out many of these small issues. In any case, Python PEP8 requires block comments to start with a '# '.

Also, you need at least two spaces before an inline comment.

Finally, maybe we could use the variable name ACC_GRAV, which is a little bit shorter and reads more the "acceleration of gravity".